setup.py

```diff
@@ -46,7 +46,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket >= 1.34.0",
+        "pytket == 1.35.0rc2",
```

Comment:
You could do 1.35.0 (please add that to the changelog)

Reply:
Done